CONTRIBUTING.md

```diff
@@ -154,6 +154,8 @@ We're also trying very hard to keep IPFS focused. This means that we might decid
 
 If your pull request is not accepted on the first try, don't be discouraged! If there's a problem with the implementation, hopefully you received feedback on what to improve.
 
+Please note: While even the smallest contributions are welcome, we collect trivial textual changes into a separate PR to reduce the number of tiny changes and to more accurately count functional contributions.
+
```

Comment:
#### Drive-by Contributions
 While even the smallest contributions are welcome, please no drive-by contributions seeking to collect airdrops.
 
 Many projects aim to reward contributors to common goods. Great. However,
 this creates an unfortunate incentive for low-effort PRs, submitted solely to
 claim rewards. These PRs consume maintainers’ time and energy to triage, with
 little to no impact on end users. If we suspect this is the intent of a PR,
 we may close it without comment. If you believe this was done in error,
 contact us via email. Reference this README section and explain why your PR
 is not a “drive-by contribution.”